src/mutator.rs

```diff
@@ -226,17 +238,22 @@ impl LocalEpoch {
 }
 
 impl Scope {
-    unsafe fn get_bag(&self) -> &mut Bag {
-        &mut *self.bag
+    #[inline]
+    unsafe fn get_bag(&self) -> Option<&mut Bag> {
```

Comment:
This can be just:

```rust
#[inline]
unsafe fn get_bag(&self) -> Option<&mut Bag> {
    self.bag.as_mut()
}
```

Reply:
Or, alternatively, delete this method and instead of calling `self.get_bag()` just write `self.bag.as_mut()` :)